### PR TITLE
Added samples for Footnotes-and-Endnotes

### PR DESCRIPTION
**Pull request by sharonaumao**
Added UG documentation samples.

- Footnotes-and-Endnotes

### CODE REVIEW

File: Footnotes-and-Endnotes/Position-for-footnote-and-endnote/.NET-Standard/Position-for-footnote-and-endnote/Program.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                paragraph.AppendText("Footnote content");
+                //Sets the footnote position
+                document.FootnotePosition = FootnotePosition.PrintImmediatelyBeneathText;
+                //Adds the new section to the document
+                section = document.AddSection();
+                //Adds a paragraph to a section
+                paragraph = section.AddParagraph();
+                //Inserts the text into the paragraph
+                paragraph.AppendText("Paragraph in Second section.");
+                //Appends the endnotes. Sets the footnote or endnote
```
Comment (Ramaraj-Marimuthu) [inline Footnotes-and-Endnotes/Position-for-footnote-and-endnote/.NET-Standard/Position-for-footnote-and-endnote/Program.cs:35]: @sharonaumao - Please modify this XML comments properly.

**Review by Ramaraj-Marimuthu — CHANGES_REQUESTED** [review]

File: Footnotes-and-Endnotes/Position-for-footnote-and-endnote/.NET-Standard/Position-for-footnote-and-endnote/Program.cs (hunk 1 above)
Comment (sharonaumao) [reply Footnotes-and-Endnotes/Position-for-footnote-and-endnote/.NET-Standard/Position-for-footnote-and-endnote/Program.cs]: @Ramaraj-Marimuthu -I made the necessary corrections

File: Footnotes-and-Endnotes/Modify-Footnote-and-Endnote-content/.NET-Standard/Modify-Footnote-and-Endnote-content/Program.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                    //Gets the paragraph at index 6
+                    WParagraph paragraph = textBody.Paragraphs[6];
+                    //Gets the footnote at index 0
+                    WFootnote footnote = paragraph.ChildEntities[0] as WFootnote;
+                    //Clear footnote content.
+                    footnote.TextBody.ChildEntities.Clear();
+                    //Add Paragraph to body of footnote.
+                    WParagraph footnoteParagraph = footnote.TextBody.AddParagraph() as WParagraph;
+                    //Sets the footnote character format.
+                    footnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
```
Comment (Ramaraj-Marimuthu) [inline Footnotes-and-Endnotes/Modify-Footnote-and-Endnote-content/.NET-Standard/Modify-Footnote-and-Endnote-content/Program.cs:28]: @sharonaumao - Is it mandatory to apply the `MarkerCharacterFormat.SubSuperScript` here?

File: Footnotes-and-Endnotes/Modify-Footnote-and-Endnote-content/.NET-Standard/Modify-Footnote-and-Endnote-content/Program.cs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                    //Append footnotes text.
+                    footnoteParagraph.AppendText(" Footnote is modified.");
+                    //Gets the textbody of the third section
+                    textBody = document.Sections[2].Body;
+                    //Gets the paragraph at index 1
+                    paragraph = textBody.Paragraphs[1];
+                    //Gets the footnote at index 0
+                    WFootnote endnote = paragraph.ChildEntities[0] as WFootnote;
+                    //Clear footnote content.
```
Comment (Ramaraj-Marimuthu) [inline Footnotes-and-Endnotes/Modify-Footnote-and-Endnote-content/.NET-Standard/Modify-Footnote-and-Endnote-content/Program.cs:37]: @sharonaumao - Please add proper XML comments here. This is not an Footnote and its Endnote.

**Review by Ramaraj-Marimuthu — CHANGES_REQUESTED** [review]

File: Footnotes-and-Endnotes/Modify-Footnote-and-Endnote-content/.NET-Standard/Modify-Footnote-and-Endnote-content/Program.cs (hunk 2 above)
Comment (sharonaumao) [reply Footnotes-and-Endnotes/Modify-Footnote-and-Endnote-content/.NET-Standard/Modify-Footnote-and-Endnote-content/Program.cs:28]: @Ramaraj-Marimuthu -Yes it is. if not then numbering format will be normal and it also carries "some hidden style with it"

File: Footnotes-and-Endnotes/Modify-Footnote-and-Endnote-content/.NET-Standard/Modify-Footnote-and-Endnote-content/Program.cs (hunk 3 above)
Comment (sharonaumao) [reply Footnotes-and-Endnotes/Modify-Footnote-and-Endnote-content/.NET-Standard/Modify-Footnote-and-Endnote-content/Program.cs:37]: @Ramaraj-Marimuthu -It is done sir

File: Footnotes-and-Endnotes/Modify-Footnote-and-Endnote-content/.NET-Standard/Modify-Footnote-and-Endnote-content/Modify-Footnote-and-Endnote-content.csproj [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RootNamespace>Modify_Footnote_and_Endnote_content</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Syncfusion.DocIO.Net.Core" Version="20.2.0.46" />
```
Comment (MohanaselvamJothi) [inline Footnotes-and-Endnotes/Modify-Footnote-and-Endnote-content/.NET-Standard/Modify-Footnote-and-Endnote-content/Modify-Footnote-and-Endnote-content.csproj:10]: Remove the version, add *

File: Footnotes-and-Endnotes/Modify-Footnote-and-Endnote-content/.NET-Standard/Modify-Footnote-and-Endnote-content/Program.cs [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+            {
+                //Loads the template document as stream
+                using (WordDocument document = new WordDocument(inputStream, FormatType.Docx))
+                {
+                    //Gets the textbody of the first section
+                    WTextBody textBody = document.Sections[0].Body;
+                    //Gets the paragraph at index 6
+                    WParagraph paragraph = textBody.Paragraphs[6];
+                    //Gets the footnote at index 0
+                    WFootnote footnote = paragraph.ChildEntities[0] as WFootnote;
```
Comment (MohanaselvamJothi) [inline Footnotes-and-Endnotes/Modify-Footnote-and-Endnote-content/.NET-Standard/Modify-Footnote-and-Endnote-content/Program.cs:22]: Access the footnote directly 
document.Footnotes[]

File: Footnotes-and-Endnotes/Modify-Footnote-and-Endnote-content/.NET-Standard/Modify-Footnote-and-Endnote-content/Program.cs [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+                    //Sets the footnote character format.
+                    footnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
+                    //Append footnotes text.
+                    footnoteParagraph.AppendText(" Footnote is modified.");
+                    //Gets the textbody of the third section
+                    textBody = document.Sections[2].Body;
+                    //Gets the paragraph at index 1
+                    paragraph = textBody.Paragraphs[1];
+                    //Gets the endnote at index 0
+                    WFootnote endnote = paragraph.ChildEntities[0] as WFootnote;
```
Comment (MohanaselvamJothi) [inline Footnotes-and-Endnotes/Modify-Footnote-and-Endnote-content/.NET-Standard/Modify-Footnote-and-Endnote-content/Program.cs:36]: Access endnote directly from WordDocument collection.

File: Footnotes-and-Endnotes/Position-for-footnote-and-endnote/.NET-Standard/Position-for-footnote-and-endnote/Program.cs [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+namespace Position_for_footnote_and_endnote
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Creates a new Word document.
+            using (WordDocument document = new WordDocument())
+            {
+				//Creates a section.
```
Comment (MohanaselvamJothi) [inline Footnotes-and-Endnotes/Position-for-footnote-and-endnote/.NET-Standard/Position-for-footnote-and-endnote/Program.cs:14]: Align the code snippet properly using VS

File: Footnotes-and-Endnotes/Position-for-footnote-and-endnote/.NET-Standard/Position-for-footnote-and-endnote/Program.cs [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+				IWSection section = document.AddSection();
```
Comment (MohanaselvamJothi) [inline Footnotes-and-Endnotes/Position-for-footnote-and-endnote/.NET-Standard/Position-for-footnote-and-endnote/Program.cs:15]: Try to use EnsureMinimal()

File: Footnotes-and-Endnotes/Position-for-footnote-and-endnote/.NET-Standard/Position-for-footnote-and-endnote/Program.cs [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+                //Adds the new section to the document
+                section = document.AddSection();
+                //Sets a section break
+                section.BreakCode = SectionBreakCode.NoBreak;
+                //Adds a paragraph to a section
+                paragraph = section.AddParagraph();
+                //Inserts the text into the paragraph
+                paragraph.AppendText("Paragraph in third Section.");
+                //Creates file stream.
+                using (FileStream outputFileStream = new FileStream(Path.GetFullPath(@"../../../Result.docx"), FileMode.Create, FileAccess.ReadWrite))
```
Comment (MohanaselvamJothi) [inline Footnotes-and-Endnotes/Position-for-footnote-and-endnote/.NET-Standard/Position-for-footnote-and-endnote/Program.cs:53]: "Sample.docx" -- output document name

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**